tasks/skylab/H5adUtils.wdl

```diff
@@ -255,16 +255,18 @@ task JoinMultiomeBarcodes {
     set -e pipefail
 
     # decompress the bgzipped fragment file
+    echo "Moving fragment file for bgzipping"
```

Comment:
Wondering why this is necessary? I've seen pushback against using "cromwell_root" before (can't recall if it was an issue for different clouds or HPC environments). Since you're moving things into the directory explicitly here it seems like it should be robust, assuming the runner has permission to write to "cromwell_root"

Reply:
It was necessary because it would not bgzip without being moved to working directory and it would not pipe to a file. I even tested it in the docker and could not get bgzip to output a separate file.